pkg/macro/cors.go

```diff
@@ -35,7 +35,7 @@ if ($request_method = 'OPTIONS') {
 }
 
 if ($cors = "true") {
-    more_set_headers 'Access-Control-Allow-Origin: $http_origin';
+    more_set_headers 'Access-Control-Allow-Origin: $origin_response';
```

Comment:
@fbomlisboa can you rename origin_response to cors_origin_response ?

Reply:
Sure! :)